### ISSUE TITLE
Add list of collaborators

### ISSUE THREAD

**Issue by simpnu** [Exodus-Privacy/website#85]
Add a list of collaborators (translation, development, etc.) somewhere on the website, for instance in the "Who" page

**Comment by VedantPol** [comment]
hey ,@pnu-s i would like to work on this issue ;)

**Comment by pnu-s** [comment]
Sure, feel free to create a PR for this issue @VedantPol !

**Comment by Jean-BaptisteC** [comment]
@pnu-s can be closed?

**Comment by pnu-s** [comment]
I don't know, I guess we still could do it, don't you think?

**Comment by Jean-BaptisteC** [comment]
It's already the case https://exodus-privacy.eu.org/fr/page/who/

**Comment by pnu-s** [comment]
No, as far as I can recall the idea was to add a list of contributors who are not active members of the organization (so 2 separate lists).

**Comment by ArnavK-09** [comment]
@pnu-s can I add this feature?/ Resolve this issue?
